scripts/check-tools.sh

```
@@ -43,11 +44,11 @@ print_status() {
   local info="$3"
 
   case "$res" in
-    0) printf "  %-12s [%s%s%s] %s\n" "$name" "$GREEN" "✅" "$RESET" "$info" ;;
-    1) printf "  %-12s [%s%s%s] %s\n" "$name" "$RED" "❌" "$RESET" "${RED}${info}${RESET}" ; return 1 ;;
-    2) printf "  %-12s [%s%s%s] %s\n" "$name" "$YELLOW" "⚠️" "$RESET" "${YELLOW}${info}${RESET}" ;;
+    0) printf "$STATUS_FORMAT" "$name" "$GREEN" "✅" "$RESET" "$info" ;;
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)